src/poliastro/twobody/propagation.py

```diff
@@ -140,23 +140,21 @@ def mean_motion(orbit, tof, **kwargs):
     p, ecc, inc, raan, argp, nu0 = rv2coe(k, r0, v0)
 
     # get the initial mean anomaly
-    M0 = nu_to_M(nu0, ecc)
+    M0 = nu_to_M(nu0 * u.rad, ecc * u.one)
```

Comment:
Instead of creating units here, why not use `nu_to_M` from `poliastro.core.angles` now? :wink: